### PR TITLE
framework/version: Redirect git stderr output

### PR DESCRIPTION
**Pull request by sdpenguin**
Add an stderr argument to redirect errors ('fatal not a git repository'
is the most common) to PIPE so that errors do not output to STDOUT.

### CODE REVIEW

File: wa/framework/version.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def get_wa_version_with_commit():
 
 def get_commit():
     p = Popen(['git', 'rev-parse', 'HEAD'],
-              cwd=os.path.dirname(__file__), stdout=PIPE)
+              cwd=os.path.dirname(__file__), stdout=PIPE, stderr=PIPE)
```
[the remark this answers is not in the document]
Comment (sdpenguin) [reply wa/framework/version.py]: Fixed, and added logging output if there is an error.

[the remark this answers is not in the document]
Comment (setrofim) [reply wa/framework/version.py]: Remove the logger-- we don't want that appearing in WA log.

[the remark this answers is not in the document]
Comment (sdpenguin) [reply wa/framework/version.py]: Done.

File: wa/framework/version.py [hunk 2]
```diff
@@ -41,8 +41,8 @@ def get_wa_version_with_commit():
 
 def get_commit():
     p = Popen(['git', 'rev-parse', 'HEAD'],
-              cwd=os.path.dirname(__file__), stdout=PIPE)
-    std, _ = p.communicate()
+              cwd=os.path.dirname(__file__), stdout=PIPE, stderr=PIPE)
+    std, stderr = p.communicate()
```
Comment (setrofim) [inline wa/framework/version.py]: Undo the change form `_` to `stderr` here, as it is not used anywhere.
